index.html

```diff
@@ -38,7 +38,133 @@ <h2>LET'S CONNECT</h2>
             </section>
         </main>
         </div>
-    </div>
-    
+        
+    </div>  
+</div>
+
+<section class="hero">
+<div class="card">
+    <img src="img/Snapshoot Portfolio.png" alt="">
+    <h1>Tonic</h1>
```

Comment:
@wolterbwambale I have observed that you are using `h1` multiple times. So like a document of text contains 1 `h1` heading. Similarly browser also looks for only one `h1` heading for one page. It actually takes it as the Title of the page for SEO purposes.
So here in these sections or work cards you can use `h2` or `h3` whatever you feel fine in your context.